### DISCUSSION TITLE
Allow postback events in links?

### DISCUSSION THREAD

**Discussion by liefra** [botfront/botfront#984] [Q&A]
Hi,

I was wondering if it is possible to make use of `postback` events in links? 

So, instead of providing a link to an URL, I would like to trigger a `postback` event, in the same way as it can be done using `quickReplies`. 

Anyone knows if this is possible? Thanks

**Comment by Abu-Ishak** [comment]
You can do it using custom action. See documentation.

**Reply by liefra** [reply]
Thank you very much for your reply. I cannot find any reference to this in the documentation.

Do you maybe have a link? Would be great

**Reply by Abu-Ishak** [reply]
You can check [here](https://botfront.io/docs/rasa/custom-actions).

**Reply by liefra** [reply]
I checked the suggested answer of @Abu-Ishak, but it is not what I was looking for.

What I try is to set a `postback` event in a normal markdown message type. So, instead of linking to an URL, the link in the message should trigger the `postback` event (without making use of Buttons and QuickReplies)
